artemis-distribution/src/main/resources/bin/artemis.cmd

```diff
@@ -25,7 +25,7 @@ set ARTEMIS_HOME="%CD%"
 POPD
 
 :CHECK_ARTEMIS_HOME
-if exist %ARTEMIS_HOME%\bin\artemis.cmd goto CHECK_JAVA
+if exist "%ARTEMIS_HOME%\bin\artemis.cmd" goto CHECK_JAVA
```

Comment:
when you create a server in windows... the profile will contain

set ARTEMIS_HOME="C:\Program Files\apache-artemis-2.5.0-SNAPSHOT"

I tried your patch and it didn't work.. for that reason I cannot merge it..


There are no issues as I see with the current thing.